money-monkey-web/app/ui/NewExpenseModal.tsx

```
@@ -38,20 +38,18 @@ export default function NewExpenseModal({
+    <div className={className}>
+      <Button Icon={MdAdd} disabled={visible} onClick={() => setVisible(true)}>
+        New Expense
+      </Button>
+      <div
+        className={
+          visible
+            ? "h-screen flex items-center justify-center fixed inset-0 bg-gray-200 bg-opacity-50"
+            : "invisible w-0 h-0"
+        }
+        onClick={() => setVisible(false)}
+      >
```

Biome

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)